certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
Sorry for being late in the game. I'm starting the work on better distribution specific abstraction in the Apache plugin in September, giving you better chance to dig in. Issues like this one are more convinient to handle after the changes.

I suggest leaving the current functionality in, and I'll ping @Jarodiv and @osirisinferi when we have something more concrete at hand. Your input is invaluable to make Certbot work flawlessly on as large amount of Gentoo installations as possible.

If you are interested in submitting a PR for the new functionality, that would be more than welcome, but unfortunately we're not quite there yet.